src/version.h

```diff
@@ -9,13 +9,13 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 70015;
+static const int PROTOCOL_VERSION = 70006;
 
 //! initial proto version, to be increased after version/verack negotiation
 static const int INIT_PROTO_VERSION = 209;
 
 //! In this version, 'getheaders' was introduced.
-static const int GETHEADERS_VERSION = 31800;
+static const int GETHEADERS_VERSION = 70005;
```

Comment:
```GETHEADERS_VERSION``` represent that version introducing ```getheaders``` into network protocol, so you should not change this value. If you want ban BitZeny 1.0, use ```MIN_PEER_PROTO_VERSION``` instead of ```GETHEADERS_VERSION```.

Reply:
OK, I understood.
Thank you.

Reply:
@rmdfwqclmrkcvpbe I want data source about that.

Reply:
@onokatio 
- https://github.com/BitzenyCoreDevelopers/bitzeny/pull/17/files/f719fa15930e0813144356c95fc7eebc98bb853b#diff-c61070c281aed6ded69036c08bd08addL17
- https://github.com/BitzenyCoreDevelopers/bitzeny/commit/341735eb8f42e898cf9d4d130709471e5d01abe2

Reply:
THX!